scripts/src/Platform/MacOS.hs

```diff
@@ -1,25 +1,28 @@
 module Platform.MacOS where
 
-import Control.Monad
-import Control.Monad.IO.Class (MonadIO, liftIO)
-import Data.FileEmbed
-import Data.List
-import Data.Monoid
-import Distribution.Simple.Utils (fromUTF8LBS)
-import System.Directory
-import System.FilePath
-import System.FilePath.Glob
-import System.IO.Temp
-import System.Process.Typed
-import System.Exit
-import Text.Printf
 
-import qualified Data.ByteString as BS
+import Prologue
 
+import qualified Data.ByteString         as BS
```

Comment:
This should just be `as ByteString`. Convention in Luna’s code is not to abbreviate like this.